src/App.tsx

```
@@ -24,355 +25,459 @@ import { getTheme } from './theme'
 
-  useEffect(() => {
-    const unsubscribeBoards = subscribeToBoards()
-    return () => unsubscribeBoards()
-  }, [subscribeToBoards, offlineMode])
+        return filtered
+    }, [cards, searchQuery, selectedLabelIds])
+
+    const cardsByColumn = useMemo(() => {
+        const grouped = filteredCards.reduce<Record<ColumnType, CardType[]>>(
+            (acc, card) => {
+                ;(acc[card.columnId] = acc[card.columnId] || []).push(card)
```

ESLint

[error] Unnecessary semicolon.

(no-extra-semi)